### PR TITLE
Add Web3Modal v2 examples

### PR DESCRIPTION
**Pull request by Juliano1612**
# Description

This adds examples using Web3Modal v2 packages ([react](https://docs.walletconnect.com/2.0/web3modal/react/installation),  [html](https://docs.walletconnect.com/2.0/web3modal/html-js/installation),  and [standalone](https://docs.walletconnect.com/2.0/web3modal/standalone/installation)) and SSX. This also updates the ssx-test-dapp to support Web3Modal v2.

Notes: 
- The standalone example doesn't support MetaMask, because [they need to update the wallet to support WalletConnect v2](https://github.com/MetaMask/metamask-mobile/issues/3957#issuecomment-1322034748). A list of wallets that supports WalletConnect v2 can be found [here](https://explorer.walletconnect.com/?type=wallet&version=2&page=1). I suggest using [Zerion](https://app.zerion.io/connect-wallet).
- **It is necessary to fill the variables in the .env files**.

# Type

- [x] Add an example

# Diligence Checklist
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by w4ll3 — APPROVED** [review]
all good and working

File: examples/ssx-test-web3modal-v2-react/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# or 
+
+npm install
+```
+
+**Run**
+
+``` bash
+yarn dev
```
Comment (obstropolos) [inline examples/ssx-test-web3modal-v2-react/README.md]: `yarn run` here - `yarn dev` isn't found

File: examples/ssx-test-web3modal-v2-standalone/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# or 
+
+npm install
+```
+
+**Run**
+
+``` bash
+yarn dev
```
Comment (obstropolos) [inline examples/ssx-test-web3modal-v2-standalone/README.md]: `yarn start` here

**Comment by obstropolos** [comment]
Additional note - need to ensure `DaoLogin` is working when using web3modalv2 - worked with -`ssx-test-dapp` where web3modal was used, but not with web3modal v2

**Review by obstropolos — APPROVED** [review]

**Pull request merged**